playbooks/roles/ocim/tasks/main.yml

```diff
@@ -0,0 +1,139 @@
+---
+- name: Set login shell for user {{ www_user }}
+  user:
+    name: "{{ www_user }}"
+    shell: /bin/bash
+
+- name: Create {{ www_data_home_dir }}
+  file:
+    path: "{{ www_data_home_dir }}"
+    owner: "{{ www_user }}"
+    group: "{{ www_group }}"
+    mode: 0750
+    state: directory
+
+- name: Copy shell configuration for user {{ www_user }}
+  template:
+    src: bashrc
+    dest: "{{ www_data_home_dir }}/.bashrc"
+    owner: "{{ www_user }}"
+    group: "{{ www_group }}"
+
+- name: Clone the opencraft repository
+  git:
+    repo: https://github.com/open-craft/opencraft.git
+    dest: "{{ opencraft_root_dir }}"
+    update: no
+  become_user: "{{ www_user }}"
+
+- name: Install the configuration/environment file
```

Comment:
I first had it in `common.yml`, but then removed it because it overwrote my `.env` file with a version that's quite useless.  The file isn't really created inside the Vagrant environment.  Since it will be created inside the repo directory, it actually ends up on the host machine, so rebuilding the devstack would nuke whatever version of `.env` you already have.

The auto-generated version of the file is rather useless anyway.  We could add a task to `cp .env.test .env` if `.env` does not exist yet, though – that's more or less equivalent to what we had before, except we don't overwrite any existing `.env` file.

Reply:
Yeah I realized how useless it is after trying to run it in common.yml myself. The `.env.test` file works just fine though on a fresh devstack, thanks!